<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionPreference"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <widget class="QMenu" name="menuDraw">
     <property name="title">
      <string>Draw</string>
     </property>
     <addaction name="action2DLayout"/>
    </widget>
    <widget class="QMenu" name="menuParaxial">
     <property name="title">
      <string>Paraxial</string>
     </property>
     <addaction name="actionParaxialRayTrace"/>
    </widget>
    <widget class="QMenu" name="menuImage_Quality">
     <property name="title">
      <string>Image Quality</string>
     </property>
     <widget class="QMenu" name="menuAberrations">
      <property name="title">
       <string>Aberrations</string>
      </property>
      <addaction name="actionTransverseRayFan"/>
      <addaction name="actionFieldCurvature"/>
      <addaction name="actionSpherochromatism"/>
      <addaction name="actionChromaticFocusShift"/>
     </widget>
     <widget class="QMenu" name="menuRays_ans_Spots">
      <property name="title">
       <string>Rays ans Spots</string>
      </property>
      <addaction name="actionSingleRayTrace"/>
      <addaction name="actionSpotDiagram"/>
     </widget>
     <widget class="QMenu" name="menuWavefront">
      <property name="title">
       <string>Wavefront</string>
      </property>
      <addaction name="actionOpdFan"/>
      <addaction name="actionWavefrontMap"/>
     </widget>
     <widget class="QMenu" name="menuPSF">
      <property name="title">
       <string>PSF</string>
      </property>
      <addaction name="actionFFT_PSF"/>
     </widget>
     <widget class="QMenu" name="menuMTF">
      <property name="title">
       <string>MTF</string>
      </property>
      <addaction name="actionGeoMTF"/>
      <addaction name="actionFFT_MTF"/>
     </widget>
     <addaction name="menuRays_ans_Spots"/>
     <addaction name="menuAberrations"/>
     <addaction name="menuWavefront"/>
     <addaction name="menuPSF"/>
     <addaction name="menuMTF"/>
    </widget>
    <addaction name="actionPrescription"/>
    <addaction name="separator"/>
    <addaction name="menuDraw"/>
    <addaction name="separator"/>
    <addaction name="menuParaxial"/>
    <addaction name="separator"/>
    <addaction name="menuImage_Quality"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSetVigFactors"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuTool"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Geopter</string>
   </property>
  </action>
  <action name="actionPreference">
   <property name="text">
    <string>Preference</string>
   </property>
  </action>
  <action name="actionDebugStream">
   <property name="text">
    <string>Debug Stream</string>
   </property>
  </action>
  <action name="action2DLayout">
   <property name="text">
    <string>2D Layout</string>
   </property>
  </action>
  <action name="actionSetVigFactors">
   <property name="text">
    <string>Set Vignetting Factors</string>
   </property>
  </action>
  <action name="actionParaxialRayTrace">
   <property name="text">
    <string>Paraxial Trace</string>
   </property>
  </action>
  <action name="actionRayAberration">
   <property name="text">
    <string>Transverse Ray Fan</string>
   </property>
  </action>
  <action name="actionPrescription">
   <property name="text">
    <string>Prescription</string>
   </property>
  </action>
  <action name="actionWavefront">
   <property name="text">
    <string>Wavefront</string>
   </property>
  </action>
  <action name="actionSingleRayTrace">
   <property name="text">
    <string>Single Ray Trace</string>
   </property>
  </action>
  <action name="actionSpotDiagram">
   <property name="text">
    <string>Standard Spot Diagram</string>
   </property>
  </action>
  <action name="actionTransverseRayFan">
   <property name="text">
    <string>Transverse Ray Fan</string>
   </property>
  </action>
  <action name="actionFieldCurvature">
   <property name="text">
    <string>Field Curvature</string>
   </property>
  </action>
  <action name="actionSpherochromatism">
   <property name="text">
    <string>Spherochromatism</string>
   </property>
  </action>
  <action name="actionChromaticFocusShift">
   <property name="text">
    <string>Chromatic Focus Shift</string>
   </property>
  </action>
  <action name="actionOpdFan">
   <property name="text">
    <string>OPD Fan</string>
   </property>
  </action>
  <action name="actionWavefrontMap">
   <property name="text">
    <string>Wavefront Map</string>
   </property>
  </action>
  <action name="actionFFT_PSF">
   <property name="text">
    <string>FFT PSF</string>
   </property>
  </action>
  <action name="actionGeoMTF">
   <property name="text">
    <string>Geometrical MTF</string>
   </property>
  </action>
  <action name="actionFFT_MTF">
   <property name="text">
    <string>FFT MTF</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
